src/browser/vdiff.js

```diff
@@ -27,7 +27,7 @@ async function ScreenshotAndCompare(opts) {
 	}
 
 	const name = this.test.fullTitle();
-	const rect = this.elem.getBoundingClientRect();
+	const rect = this.elem === document.body ? null : this.elem.getBoundingClientRect();
```

Comment:
Is the `html` tag technically a better match for what we capture here? It's just capturing the viewport, right?
```suggestion
	const rect = this.elem === document.documentElement ? null : this.elem.getBoundingClientRect();
```

Reply:
Either works! `documentElement ` is probably more accurate, but `body` easier to remember? I don't have a strong feeling, we could literally pass in 'page' as a string lol

Reply:
Ooo, I like that